src/dev/create-extension.ts

```diff
@@ -37,13 +37,39 @@ const getProjectPath = (rawArgs: string[]) => {
   return { projectPath };
 };
 
-const getChangedFilesFromFirstCommit = async (projectPath: string): Promise<string[]> => {
+const getDiffFilesFromFirstCommit = async ({
+  projectPath,
+  isDeleted = false,
+}: {
+  projectPath: string;
+  isDeleted?: boolean;
+}): Promise<string[]> => {
+  if (isDeleted) {
```

Comment:
All the logic for `isDeleted` is in one `if`, so maybe it's better to move it to a separate function?